packages/app-project/src/screens/ClassifyPage/ClassifyPageContainer.js

```diff
@@ -24,6 +25,7 @@ function ClassifyPageContainer({
   }
   if (workflowAssignmentEnabled) {
     allowedWorkflows = workflows.filter(workflow => workflow.configuration.level <= assignedWorkflowLevel)
+    allowedWorkflowID = allowedWorkflows.filter(workflow => workflow.id === workflowID)[0]?.id
```

Comment:
From reading the tests, I think `allowedWorkflowID` always ends up being set to `workflowFromUrl?.id`, which might simplify the code.

Reply:
it is indeed! that was my initial approach, but I mistakenly saw `workflows` as an empty array when testing with my staging project, and refactored accordingly.